pkg/virtctl/ssh/wrapped.go

```diff
@@ -14,11 +14,9 @@ var runCommand = func(cmd *exec.Cmd) error {
 
 func (o *SSH) runLocalCommandClient(kind, namespace, name string) error {
 
-	args := []string{}
-	args = append(args, o.buildProxyCommandOption(kind, namespace, name))
-	args = append(args, o.buildSSHTarget(kind, namespace, name))
-
-	cmd := exec.Command("ssh", args...)
+	sshArgs := fmt.Sprintf("ssh %s %s", o.buildProxyCommandOption(kind, namespace, name), o.buildSSHTarget(kind, namespace, name))
+	args := []string{"-c", sshArgs}
+	cmd := exec.Command("sh", args...)
```

Comment:
> I struggled to find the correct quoting accepted by go and the ssh. This is a possible solution but if anyone has a better fix, don't hesitate

With a simple reproducer I managed to run without using `sh`:

```go
package main

import (
	"fmt"
	"os"
	"os/exec"
)

func main() {
	args := []string{"-o", "ProxyCommand=ssh -W %h:%p root@devlab", "-i", "./id_rsa", "root@192.168.122.101", "ls -la"}
	cmd := exec.Command("ssh", args...)
	fmt.Printf("cmd: %v\n", cmd)
	cmd.Stdout = os.Stdout
	cmd.Stderr = os.Stderr
	cmd.Stdin = os.Stdin
	if err := cmd.Run(); err != nil {
		fmt.Printf("err: %v\n", err)
	}
	cmd.Wait()
}
```

I think the trick is to split all the args like `args := []string{"-o", "ProxyCommand=ssh -W %h:%p root@devlab", "-i", "./id_rsa", "root@192.168.122.101", "ls -la"}` and pass `"ProxyCommand=ssh -W %h:%p root@devlab"` as a separate element.

Reply:
@vasiliy-ul cool! Feel free to open a new PR and I'll close this

Reply:
@vasiliy-ul feel free to open a new PR and I'll close this or I can modify this accordingly. I'd also also like to avoid the use of `sh`

Reply:
I was not planning to hijack this PR :) Btw, I haven't checked the exact use-case that you were trying to fix as I do not have a quick reproducer at hand. If that is fine for you please feel free to try and apply the appraoch.

Reply:
On the contrary, many thansk for the suggestion :). I'll try your suggestion.

Reply:
I changed a bit the string parsing and now it works without the `sh`. If we separate every flag in a separate string, they are correctly read